client/src/containers/backend/People/Makers/Edit.js

```diff
@@ -25,6 +26,7 @@ class MakersEditContainer extends PureComponent {
     this.state = {
       confirmation: null
     };
+    this.updateUsers = this.updateUsers.bind(this);
```

Comment:
Note that this updateUsers function is referenced elsewhere in this component, which seems wrong.